backend/infrahub/core/node/standard.py

```
@@ -32,6 +33,9 @@
 class StandardNode(BaseModel):
     id: Optional[str] = None
     uuid: Optional[UUID] = None
+    created_by: str = Field(default=SYSTEM_USER_ID)
+    updated_by: str = Field(default=SYSTEM_USER_ID)
```

I'm not sure if this is what we want over `None`
when we actually release this, most Nodes and StandardNodes in the database will not have an `updated_by` user and I think defaulting to the SYSTEM_USER_ID in this case is kind of misleading b/c we don't actually know what user did the latest update

---

Makes sense, I've changed this. Not completely sure about the static nature of the SYSTEM_USER_ID either as we'll return a CoreAccount type within the GraphQL responses but it could be that anything with SYSTEM_USER_ID will just be a `null` value indicating the system. Alternatively we'd could populate things like display_label with system generated data so that it would always just be "system" or whatever.